src/Components/ThemeSelect/ThemeSelect.css

```
@@ -0,0 +1,37 @@
+.theme-select {
+  width: fit-content;
+  margin: 10px 10px;
```

A petty comment.  😁

Consider using a single value for the margin to simplify the code and ensure consistency. 
```suggestion
  margin: 10px;
```